src/components/FileDownload/FileDownload.tsx

```diff
@@ -26,6 +30,9 @@ export const FileDownload = ({
     <div className={classNames}>
       <a
         className="cc-file-download__link"
+        // todo: #7814
+        data-file-type={documentType}
```

Comment:
Is it worth namespacing data attributes which are concerned with analytics so they are easily distinguishable from non analytics ones? e.g. `data-analytics-file-type` or `data-tracking-file-type`

Reply:
Yep, this is exactly what I want to do... but would also require changes to:

- GTM
- D7

I've deliberately not applied namespacing to save myself the work, because I think we'd also need to undertake some investigation in to where else these specific data attributes are used...

A thought though: I could add both (namespaced and not namespaced) and create a ticket to remove? Or we could just pick up [this ticket](https://github.com/wellcometrust/corporate/issues/7814) when the migration is complete and we're not straddling two codebases?

Reply:
*also realise that ticket needs fleshing out!

Reply:
Keep the code lean for now and add these data attribute names (and any others you come across) to the opening comment of #7814 so we have a record